### ISSUE TITLE
need to click multiple times for link to work

### ISSUE THREAD

**Issue by willkg** [mozilla/rob-bugson#24]
I have this problem 80% of the time where I have to click on the "Attach this PR to a bug" link multiple times and it does nothing. Eventually it'll work. 20% of the time, it works first time.

I have no idea why this is. There's no indication in the console as to what might be happening that I can see.

**Comment by willkg** [comment]
GitHub did some UI changes, but this issue still persists.

**Comment by willkg** [comment]
The issue still persists. It's weird--sometimes a single click works and sometimes it doesn't. My wild guess is that it's my fault and related to the debounce code, but in all my tinkering with it, I haven't been able to fix it.

https://github.com/willkg/rob-bugson/blob/c4eb7c46dfd4ad6242152e961f60c7ba599cc33a/github-bugzilla-content.js#L370-L383

**Comment by chutten** [comment]
How about a semaphore?
```js
var debouncing = false;
function theListener() {
  if (debouncing) {
    console.info("Debouncing");
    return;
  }
  debouncing = true;
  // do stuff
  setTimeout(() => debouncing = false, nominalDelay);
}
```

**Comment by willkg** [comment]
Oh, I like that! I'll test that out when I get a chance. Thank you!

**Comment by willkg** [comment]
Fixed by PR #28.

Thank you, @chutten !
